standards/managing-services-interactions.md

```diff
@@ -0,0 +1,45 @@
+---
+category: Building software
+expires: 2018-03-29
+---
+# Managing interactions between services
+
+Services need to interact with one another over APIs. They need to be
+able to find one another, authenticate clients, be fault-tolerant,
+scale appropriately in line with demand, and more.
+
+## Principles
+
```

Comment:
I found this section a bit opaque. I think it would benefit a lot from some simple diagrams, and/or a description using noddy examples of the problems that might arise, and how we could mitigate against them.

Reply:
It seemed OK to me. I read it as simple characteristics of API implementation and provision, while retaining the objectives of high cohesion and low coupling.